--- jackd2-1.9.8~dfsg.4+20120529git007cdc37/debian/rules.dist	2012-11-27 21:22:31.000000000 +0100
+++ jackd2-1.9.8~dfsg.4+20120529git007cdc37/debian/rules	2014-05-20 18:38:57.275292782 +0200
@@ -41,7 +41,7 @@
 
 waf-configure-options = --prefix=/usr --classic
 waf-configure-options += --libdir=/usr/lib/$(DEB_HOST_MULTIARCH)
-waf-configure-options += $(if $(filter linux,$(DEB_HOST_ARCH_OS)),--alsa --dbus)
+waf-configure-options += $(if $(filter linux,$(DEB_HOST_ARCH_OS)),--alsa)
 waf-configure-options += $(if $(filter amd64 i386 powerpc,$(DEB_HOST_ARCH)),--firewire)
 
 DEB_MAKE_INVOKE = ./waf -v --destdir=$(CURDIR)/debian/tmp
@@ -86,7 +86,6 @@
 	dh_install -pjackd2 debian/tmp/usr/lib/$(DEB_HOST_MULTIARCH)/jack/jack_alsa.so
 	dh_install -pjackd2 debian/tmp/usr/lib/$(DEB_HOST_MULTIARCH)/jack/jack_alsarawmidi.so
 	dh_install -pjackd2 debian/tmp/usr/lib/$(DEB_HOST_MULTIARCH)/jack/audioadapter.so
-	dh_install -pjackd2 debian/tmp/usr/share/dbus-1/*
 endif	
 
 binary-install/jackd2::
